install/kubernetes/helm/istio/charts/grafana/templates/deployment.yaml

```diff
@@ -78,13 +74,27 @@ spec:
           volumeMounts:
           - name: data
             mountPath: /data/grafana
+          - name: dashboards-istio
+            mountPath: "/var/lib/grafana/dashboards/istio"
+          - name: config
+            mountPath: "/etc/grafana/provisioning/datasources/datasources.yaml"
+            subPath: datasources.yaml
+          - name: config
+            mountPath: "/etc/grafana/provisioning/dashboards/dashboardproviders.yaml"
+            subPath: dashboardproviders.yaml
       affinity:
       {{- include "nodeaffinity" . | indent 6 }}
       volumes:
+      - name: config
+        configMap:
+          name: {{ template "grafana.fullname" . }}
```

Comment:
just a general comment - I believe we had removed all the templates for `fullname` but either that wasn't the case or they snuck back into the codebase. 
```
sdake@falkor-07:~/go/src/istio.io/istio/install/kubernetes/helm/istio/charts$ grep -r template * | grep full
grafana/templates/deployment.yaml:          name: {{ template "grafana.fullname" . }}
grafana/templates/deployment.yaml:          name:  {{ template "grafana.fullname" . }}-dashboards
grafana/templates/configmap-dashboards.yaml:  name: {{ template "grafana.fullname" . }}-dashboards
grafana/templates/configmap.yaml:  name: {{ template "grafana.fullname" . }}
servicegraph/templates/ingress.yaml:  name: {{ template "servicegraph.fullname" . }}
servicegraph/templates/NOTES.txt:  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "servicegraph.fullname" . }})
servicegraph/templates/NOTES.txt:           You can watch the status of by running 'kubectl get svc -w {{ template "servicegraph.fullname" . }}'
servicegraph/templates/NOTES.txt:  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "servicegraph.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
tracing/templates/ingress.yaml:  name: {{ template "zipkin.fullname" . }}
```